01.fizzbuzz/fizzbuzz.rb

```diff
@@ -0,0 +1,21 @@
+def i_15?(i) #15の倍数か？
+    return i % 15 == 0
```

Comment:
Rubyではこういう場合`return`を書く必要はありません。最後に評価された値が戻り値となります。
```suggestion
    i % 15 == 0
```

Reply:
指摘ありがとうございます。
チェリー本でも学んだ所です。
今後は`return`をつけません。修正しました。